backend/tournesol/admin.py

```diff
@@ -177,7 +187,17 @@ def get_poll_name(self, obj):
 
 @admin.register(ComparisonCriteriaScore)
 class ComparisonCriteriaScoreAdmin(admin.ModelAdmin):
-    pass
+    list_filter = (
+        "comparison__poll__name",
+    )
+    list_display = (
+        'id',
+        'criteria',
```

Comment:
If we want to display the related comparison for each criterion. Feel free to drop this suggestion if not relevant.

```suggestion
        'comparison',
        'criteria',
```

Reply:
Fixed by https://github.com/tournesol-app/tournesol/pull/794/commits/1e87ea98145b9dbcc20e4ac51b469749b0e208de.